packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.js

```diff
@@ -64,8 +69,20 @@ export default class QuanticCategoryFacetValue extends LightningElement {
     return this.item.value;
   }
 
+  get numberOfResults() {
+    return new Intl.NumberFormat(LOCALE).format(this.item.numberOfResults);
+  }
+
   get ariaLabelValue() {
-    return `Inclusion filter on ${this.facetValue}`;
+    const labelName = I18nUtils.getLabelNameWithCount(
+      'inclusionFilter',
```

Comment:
I'd suggest to use the same casing as the custom label declaration.

```suggestion
      'InclusionFilter',
```

Reply:
this key will be used to get the right label from the labels object which is declared as the following:
```
  labels = {
     inLabel,
     inclusionFilter,
     inclusionFilter_plural,
   };
```
That's why I'm using a lower case first letter.

Reply:
Ah right, my bad :facepalm: